modules/analyzing-all-violations.adoc

```diff
@@ -11,18 +11,22 @@ By viewing the *Violations* page, you can analyze all violations and take correc
 .Procedure
 
 . In the {product-title-short} portal, click *Violations*.
-. To navigate through the different types of violations, do any of the following tasks:
-** To view the list of active violations, click the *Active* tab.
-** To view the list of resolved violations, click the *Resolved* tab. 
-** To view the list of violations that {product-title-short} attempted to resolve, click the *Attempted* tab.
-. Choose the appropriate method to view the violations from the drop-down list, which is in the upper left of the page:
-** To display the findings for application workloads, select *Applications view*.
-** To display the findings for platform components in {ocp}, select *Platform view*.
-** To display the findings for application workloads and platform components simultaneously, select *Full view*.
-. Optional: Choose the appropriate method to re-organize the information in the *Violations* page:
+. Click the tab to view violations for the following categories:
+** *User Workloads*: Displays violations for user-managed workloads.
+** *Platform*: Displays violations for workloads used by {ocp} and layered services.
+** *All violations*: Displays violations for user workload and platform components, and audit log violations for cluster resources.
+. To view different types of violations, click one of the following tabs:
+** *Active*: Displays violations in the build and deploy stages or unresolved runtime violations.
+** *Resolved*: Displays the following violations:
+*** Violations in the build and deploy stages for workloads that were removed or modified to be compliant
+*** Manually resolved runtime violations
+*** Violations in all stages that were generated before a policy exclusion was added
+** *Attempted*: Displays violations for deployment actions that were attempted but blocked by a pre-check of {product-title-short} policies by the {product-title-short} admission controller. For example, the deployment action that was attempted would have triggered a violation if it had been allowed to succeed, but the policy enforcement behavior did not allow it to succeed.
```

Comment:
blocked by evaluation of enforced policies

... but the admission controller enforcement did not allow the operation to be admitted to the cluster.